kubernetes/cray-node-discovery/values.yaml

```diff
@@ -2,10 +2,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-imagesHost: dtr.dev.cray.com:443
-
 image:
-  repository: cray/cray-node-discovery
+  repository: artifactory.algol60.net/csm-docker/stable/cray-node-discovery
+  # tag defaults to chart appVersion
```

Comment:
This comment isn't correct since tag is not an option. It can't be overridden.

Reply:
Comment removed.